api/admin/problem_details.py

```diff
@@ -373,6 +373,13 @@
     detail=_("You tried to create a new sitewide service, but a sitewide service of the same type is already configured."),
 )
 
+MULTIPLE_MARC_EXPORT_SERVICES = pd(
+    "http://librarysimplified.org/terms/problem/multiple-marc-export-services",
+    status_code=400,
+    title=_("Multiple MARC export services"),
+    detail=_("Each library can only have one MARC export service."),
+)
+
```

Comment:
I would make this more general, like "Each library can only have one service of this type".